README.md

```diff
@@ -28,7 +30,13 @@ yarn run compile
 ```
 
 -   `yarn run hydrate` creates a few auto-generated setup files locally.
--   `yarn run compile` builds source code with TypeScript
+-   `yarn run compile` builds source code to `lib/` with TypeScript
+
+> Tip: run `yarn compile -w` to keep TypeScript running in watch mode, so your output files stay up-to-date as you save source code.
+
+### Running Locally
+
+Once you've run the setup commands above, `lib/index.html` will contain a working file you can directly open in a browser locally, such as with `open lib/index.html` on Mac or `start lib/index.html` on Windows.
```

Comment:
@ayoayco does this work on your end?

Also: if there's any other info you'd want answered here, I'd love to hear it! I don't have a great sense for what other info contributors like you would want to know. ❤️

Reply:
Can confirm this works on my Mac :) Thanks @JoshuaKGoldberg!

Reply:
Opening the file in Chrome on my Mac works, but on Safari it doesn't. Maybe another bug to work on?
![Screen Shot 2022-08-29 at 7 58 44 AM](https://user-images.githubusercontent.com/4262489/187132788-2384b8ca-7009-43ca-959e-61303d1e4565.png)

Reply:
Hmm yeah that feels like a different bug